python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -114,6 +116,8 @@ def load_data_to_table(self, input_file: File) -> BaseTable:
             native_support_kwargs=self.native_support_kwargs,
             columns_names_capitalization=self.columns_names_capitalization,
             enable_native_fallback=self.enable_native_fallback,
+            databricks_job_name=f"Load data {context.get('dag_id')}_{context.get('task_id')}",
```

Comment:
Do we really need context for this?

You should just be able to do the following:

```suggestion
            databricks_job_name=f"Load data {self.dag_id}_{self.task_id}",
```